src/jit/lsraarm.cpp

```diff
@@ -762,9 +762,8 @@ void LinearScan::BuildNode(GenTree* tree)
             _snprintf_s(message, _countof(message), _TRUNCATE, "NYI: Unimplemented node type %s",
                         GenTree::OpName(tree->OperGet()));
             NYIRAW(message);
-#else
-            NYI_ARM("BuildNode default case");
```

Comment:
I would propose changing the `NYIRAW` to an assert, and then adding an `unreached()`. As this stands (unless I'm misreading), the default case will now fall through.